templates/config/initializers/sentry.rb

```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
 
-if defined?(Raven)
-  Raven.configure do |config|
-    config.processors -= [Raven::Processor::PostData]
+if defined? Sentry
+  Sentry.init do |config|
+    filter = ActiveSupport::ParameterFilter.new(Rails.application.config.filter_parameters)
+    config.before_send = lambda do |event, _hint|
+      filter.filter(event.to_hash)
+    end
```

Comment:
I propose to add the parameter filtering as a default (https://docs.sentry.io/platforms/ruby/guides/rails/configuration/options/#optional-settings)

The `filter_parameter` config is defined by default in new projects in `config/initializers/filter_parameter_logging.rb`